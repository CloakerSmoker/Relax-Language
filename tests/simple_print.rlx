#Require "$/ConsoleBase.rlx"

/*
ARGS: asdiuasodi asidohaowid sadkj blah blah
*/

define i32 Main() {
	PrintCharacters("This is a basic test", StringLength("This is a basic test"))
	
	return 0
}
