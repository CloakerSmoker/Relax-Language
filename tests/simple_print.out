This is a basic test